src/model/job_list_parameters.rs

```diff
@@ -0,0 +1,70 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct JobListParameters {
+    /// Whether to display jobs owned by all users in the project. Default False.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub all_users: Option<bool>,
+    /// The maximum number of results to return in a single response page. Leverage the page tokens to iterate through the entire collection.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub max_results: Option<u32>,
+    /// Min value for job creation time, in milliseconds since the POSIX epoch. If set, only jobs created after or at this timestamp are returned.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub min_creation_time: Option<u64>,
+    /// Max value for job creation time, in milliseconds since the POSIX epoch. If set, only jobs created before or at this timestamp are returned.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub max_creation_time: Option<u64>,
+    /// If set, show only child jobs of the specified parent. Otherwise, show all top-level jobs.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub parent_job_id: Option<String>,
+    /// Restrict information returned to a set of selected fields. Acceptable values are: full or minimal.
+    #[serde(skip_serializing_if = "Option::is_none", deserialize_with = "deserialize_projection")]
```

Comment:
Why not use an enum with two variants: full and minimal, and let Serde handle the serialization?